terraform/main.tf

```diff
@@ -63,18 +62,18 @@ resource "google_container_cluster" "app" {
 
   node_config {
     machine_type    = "n1-standard-1"
-    service_account = "${google_service_account.app_cluster.email}"
+    service_account = google_service_account.app_cluster.email
 
     oauth_scopes = [
       "https://www.googleapis.com/auth/cloud-platform",
```

Comment:
Not related, but is all we need?

Reply:
It's technically more than we need, so I'll amend this commit and work in the default IAM scopes.  Right now, there is a dedicated SA with the `any` oauth scope but no actual IAM permissions assigned to it.  It's technically very secure this way, but the cluster is most likely unable to send logs/metrics, etc.